apps/challenges/views.py

```diff
@@ -791,3 +793,11 @@ def download_all_submissions_file(request, challenge_pk, file_type):
     else:
         response_data = {'error': 'The file type requested is not valid!'}
         return Response(response_data, status=status.HTTP_400_BAD_REQUEST)
+
+
+def reload_rabbitmq_submission_worker(request):
+    if request.user.is_superuser:
+        subprocess.call("scripts/workers/restart_rabbitmq_worker.sh", shell=True)
```

Comment:
I think if we are specifying interpreter at the beginning, there is no need to add `.sh` extension to it.
Can we please drop it?